### PR TITLE
Feature/bearssl

### PR DESCRIPTION
**Pull request by felixerdy**
Adds BearSSL to WiFi Templates

### CODE REVIEW

File: templates/homev2_wifi.tpl [hunk 1]
```diff
@@ -76,7 +77,12 @@ static const uint8_t NUM_SENSORS = @@NUM_SENSORS@@;
 // Sensor SENSOR_IDs
 @@SENSOR_IDS|toProgmem@@
 
-WiFiSSLClient client;
+WiFiSSLClient sslclient;
```
Comment (mariopesch) [inline templates/homev2_wifi.tpl:80]: Should be
```
WiFiClient wifiClient;
BearSSLClient client(wifiClient);
```

File: templates/homev2_wifi.tpl [hunk 2]
```diff
@@ -26,6 +26,7 @@
 #include <VEML6070.h>
 #include <SparkFun_SCD30_Arduino_Library.h>
 #include <LTR329.h>
+#include <ArduinoBearSSL.h>
```
Comment (mariopesch) [inline templates/homev2_wifi.tpl:30]: I'm not sure if we also need to include ```#include <ArduinoECCX08.h>```

File: templates/homev2_wifi.tpl [hunk 3]
```diff
@@ -166,6 +172,10 @@ void submitValues() {
     client.stop();
     delay(1000);
   }
+
+  // check the server time for the validation of the certificate
+  ArduinoBearSSL.onGetTime(getTime);
```
Comment (mariopesch) [inline templates/homev2_wifi.tpl:177]: Should be done at the end of the setup()

**Review by mariopesch — COMMENTED** [review]
@felixerdy I've added some stuff which needs to be changed

File: templates/homev2_wifi.tpl (hunk 1 above)
Comment (felixerdy) [reply templates/homev2_wifi.tpl]: resolved in 68e5a33

File: templates/homev2_wifi.tpl (hunk 3 above)
Comment (felixerdy) [reply templates/homev2_wifi.tpl]: resolved in 982ff8c

File: templates/homev2_wifi.tpl (hunk 2 above)
Comment (felixerdy) [reply templates/homev2_wifi.tpl:30]: No need as BearSSL includes the library itself: https://github.com/arduino-libraries/ArduinoBearSSL/blob/master/src/BearSSLClient.cpp#L28

**Comment by mariopesch** [comment]
I think the test only fails because some library are not in the BSP Version 1.5.0

**Comment by mpfeil** [comment]
Compilation of `homeV2Ethernet` is failing. It´s missing `#include <NTPClient.h>`

**Pull request merged**